streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamOuterJoinTest.java

```diff
@@ -790,9 +790,7 @@ public void shouldNotEmitLeftJoinResultForAsymmetricBeforeWindow() {
             time += 1;
             inputTopic2.pipeInput(expectedKeys[2], "a" + expectedKeys[2], time);
 
-            processor.checkAndClearProcessResult(
-                new KeyValueTimestamp<>(1, "null+a1", 102L)
-            );
+            processor.checkAndClearProcessResult();
```

Comment:
Oh dear... the comment already says "should not produce any items"... Embarrassing that we still did not catch this bug originally...